safety/scan/command.py

```diff
@@ -405,32 +409,34 @@ def sort_vulns_by_score(vuln: Vulnerability) -> int:
             if file_matched_for_fix:
                 to_fix_files.append((file, to_fix_spec))
 
-            files.append(file) 
+            files.append(file)
 
     if display_apply_fix_suggestion:
         console.print()
         print_fixes_section(console, requirements_txt_found, detailed_output)
-    
+
     console.print()
     print_brief(console, ctx.obj.project, count, affected_count,
                 fixes_count)
     print_ignore_details(console, ctx.obj.project, ignored,
                          is_detailed_output=detailed_output,
                          ignored_vulns_data=ignored_vulns_data)
 
-    
+
     version = ctx.obj.schema
     metadata = ctx.obj.metadata
     telemetry = ctx.obj.telemetry
     ctx.obj.project.files = files
 
     report = ReportModel(version=version,
-                metadata=metadata, 
+                metadata=metadata,
                 telemetry=telemetry,
                 files=[],
                 projects=[ctx.obj.project])
-    
-    report_url = process_report(ctx.obj, console, report, **{**ctx.params})
+
+    exit_reason = EXIT_CODE_REASON_MAPPING.get(exit_code, "Unknown reason")
+
+    report_url = process_report(ctx.obj, console, report, exit_code=exit_code, exit_reason=exit_reason, **{**ctx.params})
```

Comment:
let's clean this old code up by replacing `**{**ctx.params}` with `**ctx.params` (i.e. remove the outer `**{}`), here and also on the very last line of this file.